plugins/tail.yaml

```diff
@@ -50,11 +50,6 @@ parameters:
     description: Include the log file path.
     type: bool
     default: true
-  - name: max_log_size
-    label: Max Log Size
-    description: The maximum size of a log entry to read before failing. Protects against reading large amounts of data into memory.
-    type: int
-    default: 1048576
```

Comment:
Should also remove `# {{$max_log_size := default 1048576 .max_log_size}}` in Set Defaults section and the `max_log_size: {{ $max_log_size }}` in file_input operator.